### PR TITLE
Fix windows pluigin service bug when no required services are provided

### PR DESCRIPTION
**Pull request by mark-kray**
Don't skip registering services and characteristics, when there are no required services provided in the sgBleConnectPeripheral method call.

(Only the plugin source code is updated, the compiled dll is unchanged)

### CODE REVIEW

**Comment by obasille** [comment]
Thank you @marko-kr , I will have a look shortly.
Just out of curiosity, may I ask for what kind of project you're using this Unity plugin?

**Comment by marko-kr1** [comment]
> Just out of curiosity, may I ask for what kind of project you're using this Unity plugin?

We're developing our own Bluetooth LE device that will integrate into our Unity app. Unfortunately I can't disclose more at the moment.

File: WinRTBle/src/LibWinRTBle/Peripheral.cpp [hunk 1]
```diff
@@ -131,11 +131,10 @@ namespace Systemic::BluetoothLE
...
                     session.MaintainConnection(maintainConnection);
 
-                    // Check required services
-                    if (requiredServices.size())
-                    {
-                        auto gattServices = servicesResult.Services();
-
+                    auto gattServices = servicesResult.Services();
+                    bool hasRequiredServices = requiredServices.size() > 0;
+                    // If no serivces are required, skip checking for missing services
```
Comment (obasille) [inline WinRTBle/src/LibWinRTBle/Peripheral.cpp]: Sorry the late review!
Please correct typo in "serivces" and leave it to singular (no "s" at the end)

Comment (marko-kr1) [reply WinRTBle/src/LibWinRTBle/Peripheral.cpp]: Apologies for the late response on my end as well. Should the plural "services" be changed to singular "service" just in the comment where the typo was made? i.e. just change
  // If no serivces are required, skip checking for missing services
  to
   // If no service is required, skip checking for missing services

Comment (obasille) [reply WinRTBle/src/LibWinRTBle/Peripheral.cpp]: Singular, thank you very much

**Review by obasille — APPROVED** [review]
Thanks so much!

**Pull request merged**